ADAL/src/cache/ios/ADKeychainTokenCache.m

```diff
@@ -373,7 +439,16 @@ - (ADTokenCacheItem*)getItemWithKey:(ADTokenCacheKey *)key
         return nil;
     }
     
-    return items.firstObject;
+    //filter out tombstone
+    if (![items.firstObject tombstone])
+    {
+        return items.firstObject;
+    }
+    else
+    {
+        [self logItem:items.firstObject message:@"Found a tombstone"];
```

Comment:
Log out the tombstone details here. We don't need to log hashes of the fields because the refreshToken should just be "<tombstone>"